.github/workflows/ci.yml

```diff
@@ -74,4 +74,25 @@ jobs:
           job-secret: ${{ secrets.JOB_SECRET }}
           region: eu-west-3
           label: ${{ steps.test-start.outputs.label }}
-          ec2-instance-id: ${{ steps.test-start.outputs.ec2-instance-id }}
```

Comment:
can you explain quickly why we get rid of it ? I'm not enough of a slab pro to see why we remove it

Reply:
It was a leftover of machulav action. I thought at first we might need it to reference existing EC2 instances but it turns out that we don't.